lms/templates/main.html

```diff
@@ -61,6 +61,9 @@
 
   <%static:css group='style-vendor'/>
   <%static:css group='${self.attr.main_css}'/>
+  <%static:css group='style-main-v2'/>
```

Comment:
@douglashall, by default lms-main-v1 will be included. This line will include lms-main-v2.